renderer/index.js

```diff
@@ -40,6 +40,7 @@ var dialog = remote.require('dialog')
 
 // For easy debugging in Developer Tools
 var state = global.state = require('./state')
+global.debugLog = require('debug')
```

Comment:
Or, were you using this to somehow enable webtorrent's debug logs in the Electron environment? I still haven't figured out how to do that.

Will remove for now, since it's not in `package.json`, but feel free to add it back.